load.Rmd

```diff
@@ -492,6 +492,7 @@ for(i in 1:nrow(cents)){
   cents$ebike_slc[i] <- sum(l$ebike_slc[j])
   cents$ebike_sic[i] <- sum(l$ebike_sic[j])
 
+  if(length(j) == 0) next
```

Comment:
If this works, go with it. j represents the home location of the centroid in l and if no lines have their origin in cents[i] then yes the loop should skip to the next one. Reasonable fix - will be less relevant when we use more lines. Thanks for asking for feedback, liking the PR workflow.